cosmosdb/sql/ps-sql-ru-update.ps1

```diff
@@ -10,7 +10,7 @@ $databaseThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/da
 $containerThroughputResourceName = $accountName + "/sql/" + $databaseName + "/" + $containerName + "/throughput"
 $containerThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/databases/containers/settings"
 $throughput = 500
-$updateResource = "database" # or "container"
```

Comment:
Changing this to `database` results in:

```
Set-AzResource : {"code":"NotFound","message":"Entity with the specified id does not exist in the system.\r\nActivityId: 5dc35c6d-90c6-457e-b46e-f82680c06cbb, Microsoft.Azure.Documents.Common/2.5.1"}
At /home/stramer/src/azure-docs-powershell-samples/cosmosdb/sql/ps-sql-ru-update.ps1:20 char:1
+ Set-AzResource -ResourceType $databaseThroughputResourceType `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : CloseError: (:) [Set-AzResource], ErrorResponseMessageException
+ FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.SetAzureResourceCmdlet
```

Reply:
When set to `container`, it looks like this is now sending a similar error to the `cosmosdb/sql/ps-container-create-index-none.ps1` error:

```
Set-AzResource : {"code":"NotFound","message":"Message: {\"code\":\"NotFound\",\"message\":\"Message: {\\\"Errors\\\":[\\\"Resource Not Found\\\"]}\\r\\nActivityId: 172525b6-dc2e-4b31-94b4-b3c2f1d7b150, Request URI: /apps/24e0c5ba-5ce4-45a1-98ff-27922cebbeee/services/52eac7ed-2f4b-40af-8b72-6f5995b75421/partitions/f9534c83-87fe-4b7a-a3e0-0a6e77d4b05d/replicas/132132990133149960s, RequestStats: \\r\\nRequestStartTime: 2019-09-18T17:37:12.5450243Z, RequestEndTime: 2019-09-18T17:37:12.5550459Z, Number of regions attempted: 1\\r\\nResponseTime: 2019-09-18T17:37:12.5550459Z, StoreResult: StorePhysicalAddress: rntbd://10.0.0.27:17000/apps/24e0c5ba-5ce4-45a1-98ff-27922cebbeee/services/52eac7ed-2f4b-40af-8b72-6f5995b75421/partitions/f9534c83-87fe-4b7a-a3e0-0a6e77d4b05d/replicas/132132990133149960s, LSN: 8, GlobalCommittedLsn: 8, PartitionKeyRangeId: , IsValid: True, StatusCode: 404, SubStatusCode: 0, RequestCharge: 1, ItemLSN: -1, SessionToken: -1#8, UsingLocalLSN: False, TransportException: null, ResourceType: Collection, OperationType: Read\\r\\nResponseTime: 2019-09-18T17:37:12.5550459Z, StoreResult: StorePhysicalAddress: rntbd://10.0.0.33:19700/apps/24e0c5ba-5ce4-45a1-98ff-27922cebbeee/services/52eac7ed-2f4b-40af-8b72-6f5995b75421/partitions/f9534c83-87fe-4b7a-a3e0-0a6e77d4b05d/replicas/132132990133149961s, LSN: 8, GlobalCommittedLsn: 8, PartitionKeyRangeId: , IsValid: True, StatusCode: 404, SubStatusCode: 0, RequestCharge: 1, ItemLSN: -1, SessionToken: -1#8, UsingLocalLSN: False, TransportException: null, ResourceType: Collection, OperationType: Read\\r\\n, SDK: Microsoft.Azure.Documents.Common/2.5.1\"}, Request URI: /dbs/database1/colls/container1, RequestStats: , SDK: Microsoft.Azure.Documents.Common/2.5.1"}
At /home/stramer/src/azure-docs-powershell-samples/cosmosdb/sql/ps-sql-ru-update.ps1:25 char:1
+ Set-AzResource -ResourceType $containerThroughputResourceType `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : CloseError: (:) [Set-AzResource], ErrorResponseMessageException
+ FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.SetAzureResourceCmdlet
```

Reply:
Explained the first issue in comment below. Added note in script for users. No explanation on second issue, could be transient. Cannot repro.